autoware_launch/launch/components/tier4_localization_component.launch.xml

```diff
@@ -4,15 +4,20 @@
   <arg name="pose_source" default="ndt" description="select pose_estimator: ndt, yabloc, eagleye"/>
   <arg name="twist_source" default="gyro_odom" description="select twist_estimator. gyro_odom, eagleye"/>
   <arg name="input_pointcloud" default="/sensing/lidar/top/pointcloud" description="The topic will be used in the localization util module"/>
+  <arg
+    name="lidar_container_name"
+    default="/sensing/lidar/top/pointcloud_preprocessor/pointcloud_container"
+    description="The target container to which lidar preprocessing nodes in localization be attached"
+  />
 
   <group>
     <include file="$(find-pkg-share tier4_localization_launch)/launch/localization.launch.xml">
-      <arg name="use_pointcloud_container" value="$(var use_pointcloud_container)"/>
       <arg name="pointcloud_container_name" value="$(var pointcloud_container_name)"/>
```

Comment:
`pointcloud_container_name` is no longer used in `launch/tier4_localization_launch/launch/localization.launch.xml`

```suggestion
```

![image](https://github.com/autowarefoundation/autoware_launch/assets/24854875/656897e1-742b-4d8a-b4ae-aad819a8c14f)

Reply:
Right, thanks. Fixed: https://github.com/autowarefoundation/autoware_launch/pull/796/commits/04ee9e57beab9d9906affe11e6db4509b4baa062